course_content/iris_course/8.Final_Exercise.ipynb

```diff
@@ -145,7 +145,9 @@
    "source": [
     "#### Part 4\n",
     "\n",
-    "Add a \"decade\" coordinate to the loaded cube using your function and the coord categorisation module."
+    "Add a \"decade\" coordinate to the loaded cube using your function and the coord categorisation module.\n",
+    "\n",
+    "(Hint: Look at the documentation for <a href='https://scitools.org.uk/iris/docs/v1.9.0/html/iris/iris/coord_categorisation.html#iris.coord_categorisation.add_categorised_coord'>iris.coord_categorisation.add_categorised_coord</a> to see how a function can be used to create a new coordinate.)"
```

Comment:
I don't think "to see how a function can be used to create a new coordinate" reads very easily.
Suggest maybe : "<link>, which can be used to create a new categorised coordinate from such a function."